Lib/fontTools/otlLib/builder.py

```diff
@@ -172,6 +172,13 @@ def setInputCoverage_(self, glyphs, subtable):
             coverage = buildCoverage(g, self.glyphMap)
             subtable.InputCoverage.append(coverage)
 
+    def setCoverage_(self, glyphs, subtable):
+        subtable.GlyphCount = len(glyphs)
+        subtable.Coverage = []
```

Comment:
```suggestion
        subtable.Coverage = [buildCoverage(g, self.glyphMap) for g in glyphs]
```

Reply:
Probably better to be consistent with existing methods (setBacktrackCoverage_/setLookAheadCoverage_/setInputCoverage_). Happy to fix them all in a separate PR.

Reply:
Sounds good (Also it would be nice if all these separate `*Count` members dropped or ignored, since they are redundant. But that is totally off-topic for this change).